core/pom.xml

```diff
@@ -7,26 +7,25 @@ SPDX-License-Identifier: AGPL-3.0-only
 <project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xmlns="http://maven.apache.org/POM/4.0.0"
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <packaging>jar</packaging>
+
+  <parent>
+    <artifactId>carbonio-user-management</artifactId>
+    <groupId>com.zextras.carbonio.user-management</groupId>
+    <version>0.1.3-1</version>
+  </parent>
+
   <artifactId>carbonio-user-management-core</artifactId>
-  <build>
-    <plugins>
-      <plugin>
-        <artifactId>maven-compiler-plugin</artifactId>
-        <configuration>
-          <source>11</source>
-          <target>11</target>
-        </configuration>
-        <groupId>org.apache.maven.plugins</groupId>
-        <version>3.6.1</version>
-      </plugin>
-    </plugins>
-    <sourceDirectory>src/main/java</sourceDirectory>
-  </build>
+  <name>carbonio-user-management-core</name>
+  <version>0.1.3-2</version>
```

Comment:
You need to change the version of this artifact in the pom.xml of the boot module